feature/gribi/otg_tests/weighted_balancing_test/setup_test.go

```diff
@@ -390,3 +431,29 @@ func debugGRIBI(t testing.TB, dut *ondatra.DUTDevice) {
 		t.Log("afts value not present")
 	}
 }
+
+// incrementedMac uses an mac string and increments it by the given i
```

Comment:
// incrementMAC increments the MAC by i.  Returns error if the mac cannot be parsed or overflows the mac address space

Reply:
Done